src/components/expanded-state/asset/ChartExpandedState.js

```diff
@@ -424,11 +433,7 @@ export default function ChartExpandedState({ asset }) {
         />
       )}
       {networks && !hasBalance && (
-        <AvailableNetworks
-          asset={assetWithPrice}
-          colors={colors}
-          networks={networks}
-        />
+        <AvailableNetworks asset={assetWithPrice} networks={networks} />
```

Comment:
i'd move this paddingBottom to this component https://github.com/rainbow-me/rainbow/pull/4109/files#diff-d5c90d0d43860ecc0a9ab4044fb284e33d3e9f80adefae3a00abe2944f76a017R133

reason for this is if we use `AvailableNetworksv2` in other place of the app it would have the padding even tho we only need it here

Reply:
moved it up in a box, using custom value since 27 isn't in the system atm, also removed the spacing in the previous component to keep things simple.